exercises/practice/sgf-parsing/.meta/example.py

```diff
@@ -47,7 +47,7 @@ def parse(input_string):
     root = None
     current = None
     stack = list(input_string)
-
+    
```

Comment:
Is this an accidental addition?

Reply:
Might be there for "readability", but not strictly necessary.  I can back it out.  😄